sig-automation/README.md

```diff
@@ -0,0 +1,137 @@
+# SIG Automation
+**Chair**: John Belamaric, Google, @johnbelamaric
+
+**Vice-chair**: Wim Henderickx, Nokia, @henderiw
+
+## Charter
+
+As agreed to by the TSC and documented in the approved [Nephio Community
+Document](https://docs.google.com/document/d/1agsjCN3aCjgPftO8AL4sCkv3Dp2ZRZ82-ikuVHAJTL8/edit?usp=sharing),
+this SIG group focuses on design and standardization of schemas, CRDs, and operators; development of CRDs, Operators, related tooling & reference Implementation; and packaging and testing of functional components of Nephio.
+
+## Subprojects
```

Comment:
Please correct me, as I recall from during the meeting mins "User Interface" e.g. GUI was planned to be part of a Sub Project. Can we reflect them in here?

Reply:
it is called User Interaction Layer and part of Package Management

Reply:
Ah, Thanks. 
In its respective "Primary Skills" sub-section can we add the desired UI framework/Technology in-plan to use?

Reply:
Agreed good point

Reply:
Ok will do so. BTW, the text above is the TSC approved scope for the SIG overall, I can't really change that language.